### PR TITLE
Fill out July 5th and May 3rd

### PR DESCRIPTION
**Pull request by u0xee**

### CODE REVIEW

File: 2021_05_03.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Twitter Space on May 3, 2021
```
Comment (ahl) [inline 2021_05_03.md:1]: Wow! From the lost archives! How did you get this recording?

**Review by ahl — COMMENTED** [review]
Legend! Thank you!

File: 2021_05_03.md (hunk 1 above)
Comment (u0xee) [reply 2021_05_03.md:1]: You're so welcome!
I was listening and hurriedly taking notes, and halfway through decided I should just turn on a recorder and enjoy listening. The next day I used the recording to take notes and [tweet them](https://twitter.com/csp_frederick/status/1389705338971049986). Then I forgot all about it until Bryan mentioned he couldn't find a copy!

Fortunately I still had the file; unfortunately there's a _lot_ of background noise. Better than nothing I guess.

Comment (ahl) [reply 2021_05_03.md:1]: **Much** better than nothing; thank you!

**Pull request merged**